web/locales/it.json

```diff
@@ -420,17 +421,17 @@
         "CK": {
             "zoneName": "Isole Cook"
         },
-        "CL-SING-SIC": {
+        "CL-SEA": {
             "countryName": "Cile",
-            "zoneName": "SING/SIC"
+            "zoneName": "Sistema de Aysén"
         },
         "CL-SEM": {
             "countryName": "Cile",
-            "zoneName": "SEM"
+            "zoneName": "Sistema de Magallanes"
         },
-        "CL-SEA": {
+        "CL-SEN": {
             "countryName": "Cile",
-            "zoneName": "SEA"
+            "zoneName": "Sistema Eléctrico Nacional"
```

Comment:
Could you please provide a source for these 3 zone names? If that's how these zones are indeed called, we might want to re-use these zone names across all other translations. cc @robertahunt

Reply:
Sure, there is a page on the Spanish wikipedia. Here's the link: https://es.wikipedia.org/wiki/Interconexi%C3%B3n_el%C3%A9ctrica_de_Chile :)

Reply:
Haha ok good point, the second sentence already explains it all:

> Hasta 2017 existían cuatro sistemas intercontectados que operaban aisladamente de los otros, pero ese año se unieron los dos principales –el Sistema Interconectado Central (SIC) y el Sistema Interconectado del Norte Grande (SING)– en el Sistema Eléctrico Nacional (SEN). Al SEN se le suman el Sistema de Aysén (SEA) y el Sistema de Magallanes (SEM), que operan en la zona austral del país.

Could you please update the english locale `en.json` with the same zone names while we're at it? Thanks!

Reply:
Done :)